services/uploads/serverless.yml

```diff
@@ -109,6 +107,9 @@ functions:
     layers:
       - !Ref ClamAvLambdaLayer
       - arn:aws:lambda:us-east-1:901920570463:layer:aws-otel-nodejs-amd64-ver-1-18-1:1
+    vpc:
+      securityGroupIds: ${self:custom.sgId}
```

Comment:
is the clamAV image inside our security group that can talk to the db for instance? I don't see why it would need to be?

Reply:
You know, there were _reasons_ when I was setting this up, but I can't recall. We can probably drop this as the av scanning lambda doesn't need to be in the VPC with our other lambdas and DB. I'll fix it up before merging!

Reply:
Oh actually, I remember now. If the lambda is not in this VPC with the server, then all the communicating traffic with the server is going to route over the public internet, which means we have to pay that egress cost. We can use VPC gateway endpoints, but we don't have access to create VPC related things in our account (CMS Cloud team has to do it for us). So I just dropped it in our VPC. 

I think we can drop the default security group though, which will keep it away from the Aurora instance.